handyrl/evaluation.py

```diff
@@ -93,7 +95,9 @@ def action(self, env, player, show=False):
 
     def observe(self, env, player, show=False):
         if self.observation:
-            _, v, _, self.hidden = self.planner.inference(env.observation(player), self.hidden)
+            outputs = self.planner.inference(env.observation(player), self.hidden)
```

Comment:
planner => model??
I think this was changed at previous update.

Reply:
Yes, this is a bug. Fixed in #68 and should be fixed in this style too.

Reply:
fixed and remove hidden state from outputs from plan()